context/app/static/js/shared-styles/icons/TooltipIcon.tsx

```diff
@@ -3,22 +3,28 @@ import { styled } from '@mui/material/styles';
 import { InfoIcon } from 'js/shared-styles/icons';
 import { SecondaryBackgroundTooltip } from '../tooltips';
 
-const StyledInfoIcon = styled(InfoIcon)(({ theme }) => ({
-  marginLeft: theme.spacing(0.5),
+interface StyledInfoIconProps {
```

Comment:
Do we want to make this prop apply to `TooltipIcon` at large?

Reply:
I'm not sure it's necessary in all cases; I'd rather have it be enabled by default as it was before and allow us to opt out in the cases where the margin is unnecessary.